servers.json

```diff
@@ -1042,7 +1042,8 @@
         "id": "minechaos",
         "name": "MineChaos",
         "addresses": [
-            "minechaos.eu"
+            "minechaos.eu",
+            "tcpshield.minechaos.eu"
```

Comment:
Please read the ip-addresses section of the README, you do not need to add sub-domains.